app/lib/actions/katello/capsule_content/create_repos.rb

```diff
@@ -35,7 +35,7 @@ def create_repo_in_pulp(capsule_content, repository)
                       content_type: repository.content_type,
                       pulp_id: repository.pulp_id,
                       name: repository.name,
-                      feed: repository.docker? ? repository.docker_feed_url(capsule_content.capsule) : repository.full_path(nil, true),
+                      feed: repository.docker? ? repository.docker_feed_url(capsule_content.capsule) : repository.full_path(capsule_content.capsule, true),
```

Comment:
If you add the schema to the `docker?` case in `full_path`, it looks like this ternary could be simplified to `repository.full_path(capsule_content.capsule, true)`